src/go/rpk/pkg/cli/cluster/selftest/start.go

```diff
@@ -48,6 +51,11 @@ of the cluster. Available tests to run:
     * Unique pairs of Redpanda nodes each act as a client and a server.
     * The test pushes as much data over the wire, within the test parameters.
 
+* Cloud tests:
+  * Latency test: 1024-bit object.
+    * Depending on read/write permissions, a series of cloud storage operations are performed.
+
```

Comment:
Can we expand what permissions/operations are performed, as a non-core user how do I set up my Redpanda cluster and storage to run these tests, and what will be tested?

Reply:
@r-vasquez IIUC the way this will work is that if the cluster is already set up with TS then these tests will use those configurations.